client/app/optionsets/common.js

```diff
@@ -64,6 +64,21 @@ export const YES_NO_UNSURE_SMALLINT = {
   },
 };
 
+export const YES_NO_UNSURE_LONGINT = {
+  YES: {
+    code: 717170000,
+    label: 'Yes',
+  },
+  NO: {
+    code: 717170001,
+    label: 'No',
+  },
+  UNSURE: {
+    code: null,
+    label: 'Unsure at this time',
+  },
+};
+
```

Comment:
@bmarchena  I am wondering why we need to make a new optionset `YES_NO_UNSURE_LONGINT` instead of reusing the `YES_NO_UNSURE` on line 37. They seem to share the same `Yes.code`, `No.code` values the CRM expects in the response.

Reply:
The problem is that the CRM is expecting a 'null' response for unsure on the eligibility question, so I added the new set rather than modify that existing one for fear of breaking functionality elsewhere

Reply:
Oh, my B. I wasn't aware that the CRM required a `null` response  for `UNSURE` rather than the  `YES_NO_UNSURE.UNSURE`value of 71717001 response.